oauth2/oauth2-client/src/main/java/org/springframework/security/oauth2/client/web/HttpSessionOAuth2AuthorizationRequestRepository.java

```diff
@@ -45,15 +55,22 @@
 
 	private final String sessionAttributeName = DEFAULT_AUTHORIZATION_REQUEST_ATTR_NAME;
 
+	private Clock clock = Clock.systemUTC();
+
+	private Duration authorizationRequestTimeToLive = Duration.ofSeconds(120);
+
+	private int maxActiveAuthorizationRequestsPerSession = 10;
+
 	@Override
 	public OAuth2AuthorizationRequest loadAuthorizationRequest(HttpServletRequest request) {
 		Assert.notNull(request, "request cannot be null");
 		String stateParameter = this.getStateParameter(request);
 		if (stateParameter == null) {
 			return null;
 		}
-		Map<String, OAuth2AuthorizationRequest> authorizationRequests = this.getAuthorizationRequests(request);
-		return authorizationRequests.get(stateParameter);
+		Map<String, OAuth2AuthorizationRequestReference> authorizationRequests = this.getAuthorizationRequests(request);
+		OAuth2AuthorizationRequestReference wrappedWithCreated = authorizationRequests.get(stateParameter);
```

Comment:
Please rename all variables that have `wrappedWith*`

Reply:
:+1: